commands/avatar.js

```diff
@@ -1,35 +1,32 @@
 exports.run = function( bot, message, args, helper, config, Rules ) {
 
 
-	Rules.apply({ needArg: false, onlyOwner: false, needAttch: false, needPermission: "EMBED_LINKS" });
-
-	if ( typeof Rules.fine() === "string" ) return message.reply( Rules.fine() );
-
-	var member = message.mentions.members.first() ? message.mentions.members.first() : message.member,
-	USTools = new helper.USTools(),
-	{ user } = member,
-
-//Loop things
-	results = [ USTools.displayGuildIcon( message.guild ), user.displayAvatarURL(null, 2048), user.displayAvatarURL(null,2048) ],
-	authors = [ message.guild.name, message.mentions.users.first() ? message.mentions.users.first().tag : false , "" ],
-	options = [ args[ 0 ] === "server" ? true : false, message.mentions.users.first() ? true : false, true ], // Server, mention, self
-	colors = [ "RANDOM", member.displayHexColor, member.displayHexColor ],
-	titles = [ "Icone de ", "Avatar de ", "Seu avatar." ],
-//
-	embed = helper.zerinhoEmbed( member ),
-	i = 0;
-
-	for ( i; i < options.length; i++ ) {
-		if ( options[ i ] ) {
-			console.log( authors[ i ] );
-			embed.setAuthor( authors[ i ], results[ i ] );
-			embed.setTitle( titles[ i ] + authors[ i ] );
-			embed.setImage( results[ i ] );
-			embed.setColor( colors[ i ] );
-
-			i = 3;
-		}
-	}
-
-	message.channel.send( embed );
+    Rules.apply({ needArg: false, onlyOwner: false, needAttch: false, needPermission: "EMBED_LINKS" });

```

Comment:
why the fuck you have to give`onlyOwner` and `needAttach` as false on every fucking command? why dont you put these parameters as already false?

Reply:
That's what i noticed!....one day after uploading every command with that...i will do something about it on the next update

Reply:
Done, now he'll need to change that too.
```suggestion
    Rules.apply({ needPermission: "EMBED_LINKS" });
```